src/thunderbots/software/ai/world/game_state.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <geom/point.h>
+#include <util/refbox_constants.h>
+
+/**
+ * @brief Holds the state of the game according to the referee
+ *
+ * @details Contains information on what period of the game it is, what type of
+ * play to run, team scores, time remaining, etc.  During normal gameplay, the
+ * information in this class is received from the [ssl
+ * refbox](https://github.com/RoboCup-SSL/ssl-refbox)
+ * program over the network.
+ *
+ * Taken from
+ */
+class GameState
+{
+   public:
+    enum State
+    {
+        HALT,    // Robots must not move
+        STOP,    // Robots must stay 500mm away from ball
+        SETUP,   // Robots not on starting team must stay 500mm away from ball
+        READY,   // A robot on the starting team may kick the ball
+        PLAYING  // Normal play
+    };
+
+    // Types of restarts
+    enum Restart
+    {
+        NONE,
+        KICKOFF,
+        DIRECT,
+        INDIRECT,
+        PENALTY,
+        BALL_PLACEMENT
+    };
```

Comment:
Yeah, it's a subset of `Command` enum items that's updated based on the `Command` state transitions

Reply:
@jordanzliu I think it would be good to add a comment above these enums that explains that they are subsets of the `Command` enum that combine in different ways to represent all the `RefBoxGameStates`